HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/APIResource.swift

```diff
@@ -46,9 +46,9 @@ struct APIResource<T: Decodable>: Resource {
     
     var apiVersion: Int {
         switch domain {
-        case .default: return 3
+        case .default: return 4
         case .gym: return 2
```

Comment:
[Request]: @razvancapra we need to remove `gym` from `HealthAPILibrary` and `HealthSDK` resources